simapp/export.go

```diff
@@ -93,8 +93,15 @@ func (app *SimApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []
 	}
 
 	for _, delegation := range dels {
-		valAddr:= sdk.MustValAddressFromBech32(delegation.ValidatorAddress)
-		delAddr := sdk.MustAccAddressFromBech32(delegation.DelegatorAddress)
+		valAddr, err := app.InterfaceRegistry().SigningContext().ValidatorAddressCodec().StringToBytes(delegation.ValidatorAddress)
```

Comment:
lots of concatenated pointers here, maybe something compact can be done to check all ptrs in between ?

Reply:
not sure if necessary as the func will panic anyways